src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/move/MovePersister.kt

```diff
@@ -132,7 +131,7 @@ class MovePersister(
       state = JourneyState.cancelled,
       vehicleRegistration = null,
       notes = "FAKE JOURNEY ADDED FOR CANCELLED BILLABLE MOVE",
-      effectiveYear = effectiveYearForDate(move.moveDate ?: LocalDate.now()),
+      effectiveYear = effectiveYearForDate(move.moveDate ?: timeSource.date()),
```

Comment:
Spotted this on my travels.  Should be using the TimeSource when working out system date times.  This also makes it testable.